chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+
+from chaosgcp import client
+
```

Comment:
```suggestion
from typing import Any, Dict
from chaoslib.types import Configuration, Secrets
from chaosgcp import client
```

removing unnecessary spaces